routers/match.py

```
@@ -0,0 +1,22 @@
+from datetime import datetime
+from fastapi import HTTPException
+
+@router.get("/matches")
```

Ruff (0.11.9)

Undefined name `router`

(F821)